GTFS.DB.SQLite/Collections/SQLiteRouteCollection.cs

```diff
@@ -193,7 +193,7 @@ public IEnumerable<Route> Get()
                     ShortName = x.IsDBNull(2) ? null : x.GetString(2),
                     LongName = x.IsDBNull(3) ? null : x.GetString(3),
                     Description = x.IsDBNull(4) ? null : x.GetString(4),
-                    Type = (RouteTypeExtended)x.GetInt64(5),
+                    Type = (RouteType)x.GetInt64(5),
```

Comment:
I think you have to change this in the in-memory and postres database collections as well.

Reply:
Those seem to have been deleted from the `develop` branch. They're in master though?

Reply:
MM, I'm on master on my local. So maybe master is more recent than develop?

Reply:
master is the default branch in this repo

Reply:
or at least we're using master and not develop

Reply:
I'm working on fixing this

Reply:
This should all be fixed now.
I merged in master, fixed the postgres db collections and changed the base branch for this PR to master.